spec/models/odk/response_saver_spec.rb

```diff
@@ -55,28 +50,20 @@
         end
 
         thread2 = Thread.new do
-          # wait until the first thread is sleepin
-          sleep(3)
```

Comment:
I removed the sleep here because when I was writing the new specs in the other file, it turned out not to be necessary -- both threads running simultaneously will try to write to the database at the same time, throwing the error we want.

Reply:
ah ok, good!